quantum/quantum.h

```diff
@@ -135,70 +133,59 @@ extern uint32_t default_layer_state;
     #include "hd44780.h"
 #endif
 
-//Function substitutions to ease GPIO manipulation
-#ifdef __AVR__
-    #define PIN_ADDRESS(p, offset) _SFR_IO8(ADDRESS_BASE + (p >> PORT_SHIFTER) + offset)
-
-    #define pin_t uint8_t
-    #define setPinInput(pin) PIN_ADDRESS(pin, 1) &= ~ _BV(pin & 0xF)
-    #define setPinInputHigh(pin) ({\
-            PIN_ADDRESS(pin, 1) &= ~ _BV(pin & 0xF);\
-            PIN_ADDRESS(pin, 2) |=   _BV(pin & 0xF);\
-            })
-    #define setPinInputLow(pin) _Static_assert(0, "AVR Processors cannot impliment an input as pull low")
-    #define setPinOutput(pin) PIN_ADDRESS(pin, 1) |= _BV(pin & 0xF)
-
-    #define writePinHigh(pin) PIN_ADDRESS(pin, 2) |=  _BV(pin & 0xF)
-    #define writePinLow(pin) PIN_ADDRESS(pin, 2) &= ~_BV(pin & 0xF)
-    static inline void writePin(pin_t pin, uint8_t level){
```

Comment:
This inline function has been replaced by a macro.

https://github.com/qmk/qmk_firmware/pull/5021/files#diff-f3b139fb3777a376a7436fcb96b8dc1dR164